src/MUSI6106Exec/MUSI6106Exec.cpp

```diff
@@ -16,33 +19,61 @@ void    showClInfo ();
 // main function
 int main(int argc, char* argv[])
 {
-    std::string             sInputFilePath,                 //!< file paths
-                            sOutputFilePath;
+    std::string sInputFilePath,                 //!< file paths
+                sOutputFilePath;
 
-    static const int        kBlockSize = 1024;
+    static const int kBlockSize = 1024;
 
-    clock_t                 time = 0;
+    clock_t time = 0;
 
-    float                   **ppfAudioData = 0;
+    float **ppfAudioData = 0;
+    float **ppfFilterData = 0;
 
-    CAudioFileIf            *phAudioFile = 0;
-    std::fstream            hOutputFile;
+    CAudioFileIf *phAudioFile = 0;
+    CCombFilter *phFilterFile = 0;
+    std::fstream hOutputFile;
     CAudioFileIf::FileSpec_t stFileSpec;
+    
+    std::string sFilterType;
+    float fGain = 0;
+    float fDelay = 0;
 
     showClInfo();
 
     //////////////////////////////////////////////////////////////////////////////
     // parse command line arguments
-    if (argc < 2)
+    // input example: [exe path type gain delay]
```

Comment:
The tests should be implemented in this case when there are no arguments passed when executing

Reply:
Also not sure if I'm missing anything, but I can't see any test functions implemented or called